platform/wisdomctl/wisdomctl.py

```diff
@@ -0,0 +1,31 @@
+from ocifs import OCIFileSystem
+
+import click
+
+@click.group()
+def cli(): pass
+
+REMOTE = click.argument("remote")
+
+MB = 1024 ** 2
+def slowCopy(src, dest):
+    while True:
+        hunk = src.read(MB)
+        if not hunk: break
+        dest.write(hunk)
+
+@cli.command()
+@REMOTE
+@click.argument("local", type=click.File("wb"))
+def getFromSource(remote, local):
```

Comment:
getfromSoure is global. Should it not be `GetFromSource()`?

Reply:
Nah, Python doesn't mind. Click lowercases the entire name either way. I think it should be possible to make Click render these as "get-from-source" and "send-to-source", which would be easier to read.